src/Ng/SearchIndexFromCatalogCollector.cs

```diff
@@ -53,7 +53,7 @@ public SearchIndexFromCatalogCollector(Uri index, Lucene.Net.Store.Directory dir
 
                 indexWriter.ExpungeDeletes();
 
-                indexWriter.Commit(CreateCommitMetadata(commitTimeStamp));
+                indexWriter.Commit(DocumentCreator.CreateCommitMetadata(commitTimeStamp, "from catalog", Guid.NewGuid().ToString()));
```

Comment:
Good point, I missed the `commit-time-stamp` property which is used in one place of our code.